core/templates/dev/head/pages/story_editor/main_editor/story_node_editor_directive.html

```diff
@@ -0,0 +1,80 @@
+<div class="node-field">
+  <label> Node Outline </label>
+  <div style="float: right;">
+     Outline Finalized
+    <i ng-if="!isOutlineFinalized()" ng-click="finalizeOutline()" class="material-icons md-18">check_box_outline_blank</i>
+    <i ng-if="isOutlineFinalized()" ng-click="unfinalizeOutline()" class="material-icons md-18">check_box</i>
+  </div>
+  <div ng-if="!outlineEditorIsShown">
+    <angular-html-bind html-data="editableOutline">
+    </angular-html-bind>
+    <button type="button"
+            class="btn btn-default save-button"
+            ng-click="closePreviewOutline(editableOutline)">
+      Edit
+    </button>
+  </div>
+  <div ng-if="outlineEditorIsShown">
+    <schema-based-editor schema="OUTLINE_SCHEMA"
+                         local-value="editableOutline">
+    </schema-based-editor>
+    <br>
```

Comment:
Don't use br for styling. Use CSS (margins etc.) instead.

(The page needs to make sense semantically for users with screenreaders.)